modules/payloads/stages/windows/upexec.rb

```diff
@@ -68,12 +68,7 @@ def initialize(info = {})
   #
   def handle_connection_stage(conn, opts={})
     begin
-      # bug fix for: data = ::IO.read(datastore['PEXEC'])
-      # the above does not return the entire contents
```

Comment:
👀  👀  👀

Reply:
lol. perhaps we should add a rubocop rule for this?